bin/gpm

```diff
@@ -53,6 +54,12 @@ if (destOpt.value) {
     destOpt.value = Path.resolve(destOpt.value.replace("~", Ul.home()));
 }
 
+if (inputOpt.value === ".") {
+    // Load a package.json file in the current directory
+    inputOpt.value = ReadJson(Path.join(process.cwd(), "package.json"));
+    destOpt.value = destOpt.value || Path.join(process.cwd(), "node_modules");
```

Comment:
Almost there!

```js
destOpt.value = Deffy(destOpt.value, "node_modules");
```

And move this `if` before the previous one (line 53). That will help us reusing the same code.